### PR TITLE
[Programs] Hacktoberfest: Use generic Hacktoberfest logo for program card

### LINKED ISSUE #3224: [Programs] Hacktoberfest: Use generic Hacktoberfest logo for program card
#### Description
The [programs page](https://layer5.io/careers/programs) has a 2022 version of Layer5's Hacktoberfest participation. The Hacktoberfest event design changes from year to year.

#### Expected Behavior
The Hacktoberfest program card should use generic Hacktoberfest logo to represent all years that Layer5 has participated.

#### Screenshots
Screen Shot 2022-09-30 at 4 59 12 PM

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by XDRAGON2002**
Signed-off-by: Anant Vijay <anantvijay3@gmail.com>

**Description**

This PR fixes #3224 

**Notes for Reviewers**
Changed the logo to be used on events card

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0924794ada9b60a33e9cce24b429491cace1af2b at: https://63378872f4edc0004966f6ff--layer5.netlify.app

**Comment by XDRAGON2002** [comment]
So I am facing an interesting situation with this one here, I made the change (which I think is in the correct place and have also referenced it from the initial change from the 2021 logo to the 2022 logo as evident in #3199) but on local it breaks the build, and as visible from the preview link above, the changes have not taken place.
Could someone help point out what I am missing here?

**Comment by Nikhil-Ladha** [comment]
The file where you have made the change is: https://63378872f4edc0004966f6ff--layer5.netlify.app/community/handbook/mentorship-programs

Just update the CSS to center align the image vertically, the rest looks fine.

For the programs page, the code rendering the page is: https://github.com/layer5io/layer5/blob/cba9ea9505ded98f1acf42b4f9efb72aadceb79b/src/pages/careers/programs.js

Based on the way we select the image to display for each program on the programs page, I think we don't require a change there. Since it will automatically pick up the image of the latest program that we are showing. So, if there is an update to the image in the latest program, the programs page will be having that updated image.

Though, a better change would be to move the generic "Hactoberfest" image to the "assets/images/programs" folder.

**Comment by l5io** [comment]
🚀 Preview for commit f234adf1a079edb420979680a82998956ed963bf at: https://6337f7900dfefb44b42a1ec5--layer5.netlify.app

File: src/sections/Community/Handbook/mentorships.js [hunk 1]
```diff
@@ -32,6 +32,10 @@ const ConductWrapper = styled.div`
           display: block;
           margin-left: auto;
           margin-right: auto;
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/mentorships.js:38]: Why go so far as to just center an image? Moreover, this creates issues in mobile view.
Use `align-self: center` on the `Col` component having the img i.e, use the `id` attribute of the component to apply CSS to all the `Col` having the img as a child in this page.

Also, after doing so remove the `padding-top` CSS applied to the img at line: 29

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
No need to update the `program.js` file.
Just updating the handbook is enough.

File: src/sections/Community/Handbook/mentorships.js (hunk 1 above)
Comment (XDRAGON2002) [reply src/sections/Community/Handbook/mentorships.js]: Thanks for the suggestions!

**Comment by l5io** [comment]
🚀 Preview for commit 27dce39965a71f510e23d94e874a46090bf178af at: https://633815b7bf4c255824a34050--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
lgtm

**Pull request merged**